### PR TITLE
Create a copie for portfolio mobile version

### PR DESCRIPTION
**Pull request by Baroka-wp**
- [ ] Create an empty repo
- [ ] Copy the files 
- [ ] Config linters 
- [ ]  Pull request

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -16,11 +16,6 @@ h2 {
   padding: 0;
 }
 
-h3 {
-  margin: 0;
-  padding: 0;
-}
-
 p {
```
Comment (Baroka-wp) [inline css/style.css:19]: - [ ] Please do not delete his specification. It is important to normalize your page

File: index.html [hunk 2]
```diff
@@ -65,7 +65,7 @@ <h2>Projects</h2>
       <article class="project">
         <img class="project-1-img" src="images/project-img-4.png" alt="project 1imgage">
         <section class="rectangle">
-          <p>Project Tinder clone</p>
+          <p>Project Tinders clone</p>
           <ul class="technology-tag">
```
Comment (Baroka-wp) [inline index.html:69]: - [ ] Kindly add the absolute link for your image to avoid any future concerns when image is not in the same repos

**Review by Baroka-wp — COMMENTED** [review]
# Changes required ♻️ 
you already done all assigment. Thank you for your hard work.
Just some little think you add for improving your code. 



- [ ]  Kindly add the normalize style for you H3
- [ ] kindly add the absolute link for images 

Thank you

**Review by MeqdamAlqudah — APPROVED** [review]
# Hi @Baroka-wp , 

### Status: Approved ✔️

:+1:  You have implemented all the suggested changes made by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**_Check the comments under the review_**


**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Dipesh Kumar_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**